prepare_data.sh

```diff
@@ -6,17 +6,28 @@ INPUT_SHP_ARCHIVE=${DATASET_PREFIX}_2018-02-16.tgz
 INPUT_SHP_FILE=export/routegeometry.shp
 OUTPUT_MBTILES=${DATASET_PREFIX}_LV95_WM.mbtiles
 
+POI_DATASET_PREFIX=data/station-didok
+
 echo "Convert into GEOJSON LV95 from LV03 shapefile archive..."
 ogr2ogr -f "GeoJSON" ${DATASET_PREFIX}_LV95.geojson /vsitar/${INPUT_SHP_ARCHIVE}/${INPUT_SHP_FILE} -s_srs EPSG:21781 -t_srs EPSG:2056 -select land,route_id,surface
 
 echo "Run python script to convert from LV95 to 'Web Swiss' (fake web mercator)..."
 python3 lv95_web_swiss/lv95_web_swiss.py ${DATASET_PREFIX}_LV95.geojson ${DATASET_PREFIX}_web_swiss_wrong_crs.geojson
 
+echo "Converting POI SBB to projection LV95..."
+ogr2ogr -f "GeoJSON" -s_srs EPSG:4326 -t_srs EPSG:2056 ${POI_DATASET_PREFIX}-lv95.geojson
+
+echo "Run python script on POI SBB to convert from LV95 to 'Web Swiss' (fake web mercator)..."
+python3 lv95_web_swiss/lv95_web_swiss.py ${POI_DATASET_PREFIX}-lv95.geojson ${POI_DATASET_PREFIX}-lv95-web-swiss.geojson ${DATASET_PREFIX}_web_swiss_wrong_crs.geojson
 # Change header of file to make it look like web mercator
 ogr2ogr -f "GeoJSON" ${DATASET_PREFIX}_web_swiss.geojson -a_srs EPSG:3857 ${DATASET_PREFIX}_web_swiss_wrong_crs.geojson
```

Comment:
Is this not also required for the POI dataset? Maybe you can integrate it into the python script, so you don't have to do it here repeatedly :+1:

Reply:
This is a bit more tricky.
This line takes time only to change the header file telling which projection is used. It avoids a warning from tippecanoe, but is not mandatory.
If tippecanoe changes its policy regarding the projection specification and throws an error instead of a warning, this line would be needed.
I will comment it out, since I do not see the utility of this line now and it can lead to confusion if the file is read by something else.